<ui version="4.0" >
 <class>Merge</class>
 <widget class="QDialog" name="Merge" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Merge</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBoxWc" >
     <property name="title" >
      <string>Working Copy</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="labelWcPath" >
          <property name="text" >
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editWcPath" />
        </item>
        <item>
         <widget class="QPushButton" name="buttonWcPath" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxChanges" >
     <property name="title" >
      <string>Changes</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBoxTo" >
        <property name="title" >
         <string>To</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="editToRevision" />
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="editToUrl" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelToRevision" >
           <property name="text" >
            <string>Revision</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="labelToURL" >
           <property name="text" >
            <string>URL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBoxFrom" >
        <property name="title" >
         <string>From</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="editFromRevision" />
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="editFromUrl" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelFromRevision" >
           <property name="text" >
            <string>Revision</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="labelFromURL" >
           <property name="text" >
            <string>URL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Merge</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Merge</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
